src/js-marketing-notification.test.ts

```diff
@@ -86,8 +68,42 @@ QUnit.test('Should toggle expanded class when clicked', assert => {
   assert.ok(node.classList.contains('expanded'));
 });
 
-QUnit.test('Should add to local storage after initial load', assert=> {
+QUnit.test('should add to local storage after initial load', assert=> {
   notifier.output(notification);
 
   assert.notEqual(lsm.getItem('catalog-request'), null);
 });
+
+QUnit.test('should contain tab content', assert => {
+  notifier.output(notification);
+  const node = document.querySelector('.marketing-notification__tab');
+
+  assert.notEqual(node, null);
+  assert.notEqual(node.querySelector('h2'), null);
+});
+
+QUnit.test('should contain notification block', assert => {
+  notifier.output(notification);
+  const node = document.querySelector('.marketing-notification__body');
+
+  assert.notEqual(node, null);
+  assert.notEqual(node.querySelector('a'), null);
+});
+
+QUnit.test('should add tab and body classNames', assert => {
+  const classNames = ['sample', 'sample2'];
+  notification.tabContent.classNames = [];
+  notification.notificationBlocks.forEach(block => block.classNames = []);
+  classNames.forEach(className => {
+    notification.tabContent.classNames.push(className);
+    notification.notificationBlocks.forEach(block => block.classNames.push(className));
+  });
+  notifier.output(notification);
+  const tabNode = document.querySelector('.marketing-notification__tab'),
+        bodyNodes = document.querySelectorAll('.marketing-notification__block');
+
+  assert.ok(classNames.every(className => tabNode.classList.contains(className)));
+  assert.ok(classNames.every(className => {
```

Comment:
Would it possibly be better to run a loop and do each `assert` inside of it? I feel like this would be easier to read.